projects/igniteui-angular/migrations/update-11_1_0/changes/outputs.json

```diff
@@ -320,6 +320,46 @@
                 "selector": "igx-chip",
                 "type": "component"
             }
+        },
```

Comment:
If that pr is targeting master (12.0.x)v, we have to move those migration changes.